api/model/src/main/java/org/projectnessie/model/IcebergContent.java

```diff
@@ -15,4 +15,8 @@
  */
 package org.projectnessie.model;
 
-public abstract class IcebergContent extends Content {}
+public abstract class IcebergContent extends Content {
+  public abstract String getMetadataLocation();
+
+  public abstract long getVersionId();
```

Comment:
Isn't this method addition breaking binary backwards compatibility?

Reply:
Also: don't you need to modify `nessie-model-quarkus` accordingly?

Reply:
How would an addition break backwards compatibility in Java?

BTW: `nessie-model-quarkus` uses the same sources

Reply:
Because it's an abstract method, thus requiring code extending this class to implement it.

Reply:
Nothing except the types in `:nessie-model` extend `IcebergContent`.